src/txn/txn.c

```diff
@@ -2367,7 +2367,7 @@ __wt_txn_is_blocking(WT_SESSION_IMPL *session)
      * Check if either the transaction's ID or its pinned ID is equal to the oldest transaction ID.
      */
     return (txn_shared->id == global_oldest || txn_shared->pinned_id == global_oldest ?
-        __wt_txn_rollback_required(session, WT_TXN_ROLLBACK_REASON_CACHE) :
+        __wt_txn_rollback_required(session, WT_TXN_ROLLBACK_REASON_OLDEST_FOR_EVICTION) :
```

Comment:
As commented above, let's keep the former naming.

Reply:
addressed here [f789e91](https://github.com/wiredtiger/wiredtiger/pull/8665/commits/f789e9155171960af9babea55760494256cc3ff7)